test/unit/ngsiv2/examples/contextRequests/updateContextHiddenAttrsPlugin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "location": {
+    "type": "geo:json",
+    "value": {
+      "coordinates": [
+        9.704217,
+        50.5744
+      ],
+      "type": "Point"
+    }
+  },
+  "TimeInstant": {
+    "type": "DateTime",
+    "value": "2021-11-18T14:20:14.901Z"
+  }
+}
```

Comment:
I am not able to see where is used this file, can you confirm me it is actually used?

Reply:
Fixed - deleted 7fecd2e